spec/std/random/system_spec.cr

```diff
@@ -0,0 +1,10 @@
+require "spec"
+require "random/system"
+
+describe "Random::System" do
+  rng = Random::System.new
+
```

Comment:
I think it's worth testing one nontrivial case, like generating a 64 bit signed integer. It is not part of `Random::System` directly but having this called in real code can be better at pointing the compiler at something that accidentally went wrong.